osu.Game/Screens/OnlinePlay/Playlists/PlaylistsResultsScreen.cs

```diff
@@ -182,7 +186,23 @@ private APIRequest createIndexRequest(Action<IEnumerable<ScoreInfo>> scoresCallb
         /// <param name="pivot">An optional pivot around which the scores were retrieved.</param>
         private void performSuccessCallback([NotNull] Action<IEnumerable<ScoreInfo>> callback, [NotNull] List<MultiplayerScore> scores, [CanBeNull] MultiplayerScores pivot = null) => Schedule(() =>
         {
-            var scoreInfos = scoreManager.OrderByTotalScore(scores.Select(s => s.CreateScoreInfo(scoreManager, rulesets, playlistItem, Beatmap.Value.BeatmapInfo))).ToArray();
+            var scoreInfos = scores.Select(s => s.CreateScoreInfo(scoreManager, rulesets, playlistItem, Beatmap.Value.BeatmapInfo)).ToArray();
+
+            if (multiplayerClient?.Room?.Settings.NoScoreMultiplier == true)
+            {
+                // recalculate score without score multiplier
+                var ruleset = Ruleset.Value.CreateInstance();
+                var scoreProcessor = ruleset.CreateScoreProcessor();
+                scoreProcessor.ScoreMultiplier = 1;
+
+                foreach (var scoreInfo in scoreInfos)
+                {
+                    scoreInfo.IsScoreMultiplierDisabled = true;
+                    scoreInfo.TotalScore = scoreProcessor.ComputeScore(Rulesets.Scoring.ScoringMode.Standardised, scoreInfo);
+                }
+            }
+
+            scoreInfos = scoreManager.OrderByTotalScore(scoreInfos).ToArray();
```

Comment:
Why does this recalculate each score manually, only to then call `OrderByTotalScore()` _again_?

Reply:
I remember there was a reason for it, but it is no longer needed with the changes i did.